include/psp2/gxm.h

```diff
@@ -48,52 +47,65 @@ typedef enum SceGxmErrorCode {
 
 typedef void (SceGxmDisplayQueueCallback)(const void *callbackData);
 
+typedef enum SceGxmInitializeFlags {
+    SCE_GXM_INITIALIZE_FLAG_DEFAULT         = 0x00000000u, //!< Default initialization flag.
+    SCE_GXM_INITIALIZE_FLAG_PB_LPDDR        = 0x00000001u, //!< Allocate the Parameter Buffer from MAIN LPDDR instead of CDRAM.
+    SCE_GXM_INITIALIZE_FLAG_SHARED_SYNC     = 0x00000002u, //!< Enable support for shared sync objects.
+    SCE_GXM_INITIALIZE_FLAG_SHAREDPB_CREATE = 0x00000004u, //!< Create a shared parameter buffer.
+    SCE_GXM_INITIALIZE_FLAG_SHAREDPB_OPEN   = 0x00000008u, //!< Open a shared parameter buffer. Provided parameterBufferSize will function as a minimum required size.
+    SCE_GXM_INITIALIZE_FLAG_EXTENDED_FORMAT = 0x00000010u  //!< Enable support for extended texture/color/pixel formats
+} SceGxmInitializeFlags;
+
 typedef struct SceGxmInitializeParams {
-	unsigned int flags;
-	unsigned int displayQueueMaxPendingCount;
-	SceGxmDisplayQueueCallback *displayQueueCallback;
-	unsigned int displayQueueCallbackDataSize;
-	SceSize parameterBufferSize;
+	unsigned int flags;                               //!< One or more ::SceGxmInitializeFlags.
```

Comment:
re-type it as SceGxmInitializeFlags maybe?

Reply:
Enum is guaranteed to be 32-bit?

Reply:
Hm, no. Not in this case. It should be big enough to hold maximum enum value.
Actually, fuck it, enums doesn't seem to be typechecked in C anyway